### ISSUE TITLE
[ClearLag] Plugin command (/lagg)

### ISSUE THREAD

**Issue by GalaxyEaterGR** [VolmitSoftware/React#213]
Please remove the command (/lagg) from the react because i'm using the ClearLag plugin on my server too and i can not use the plugin because of that or please add a detection that can detect if the server is using the plugin ClearLag.

**Comment by cyberpwnn** [comment]
I dont see lagg anywhere
```yml
name: ${react.plugin.name}
version: ${project.version}
main: ${react.plugin.main}
author: cyberpwn
description: React is a smart performance plugin designed to automatically fix lag from the source.
load: STARTUP
website: https://www.spigotmc.org/resources/react-smart-server-performance.21057/
database: false
softdepend: [ProtocolLib, FastAsyncWorldEdit]
prefix: React
commands: 
  react: 
    aliases: [re, ract]
  rai:
    aliases: [react-ai]
```

Just in case i also searched the entire project and there were 0 results for "lagg". What is the issue?

**Comment by cyberpwnn** [comment]
Fixed in 6.0.6
